pkg/workload/schemachange/operation_generator.go

```
@@ -1930,6 +1985,13 @@ func (og *operationGenerator) setColumnNotNull(tx *pgx.Tx) (string, error) {
 		}
 	}
 
+	hasPKSchemaChanges, err := tableHasOngoingAlterPKSchemaChanges(tx, tableName)
+	if err != nil || hasPKSchemaChanges {
```

should we return the error if something errored?